src/vs/workbench/contrib/search/browser/searchResultsView.ts

```diff
@@ -244,6 +244,11 @@ export class FileMatchRenderer extends Disposable implements ICompressibleTreeRe
 		templateData.badge.setTitleFormat(count > 1 ? nls.localize('searchMatches', "{0} matches found", count) : nls.localize('searchMatch', "{0} match found", count));
 
 		templateData.actions.context = <ISearchActionContext>{ viewer: this.searchView.getControl(), element: fileMatch };
+
+		// when hidesExplorerArrows: true, then the file nodes should still have a twistie because it would otherwise
+		// be hard to tell whether the node is collapsed or expanded.
+		const twistieContainer = templateData.el.parentElement?.parentElement?.querySelector('.monaco-tl-twistie');
```

Comment:
This looks pretty fragile as you're reaching into the impl or the list view. Is this how the explorer does it?

Reply:
Yeah, that's how it does it. It does note that it's a bit hacky, though.
https://github.com/microsoft/vscode/blob/9ac614710c23b343c90ee32c8d6c77606eb7888f/src/vs/workbench/contrib/files/browser/views/explorerViewer.ts#L409